tools/lib/src/integration_test_command.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: implementation_imports
+
+import 'package:file/file.dart';
+import 'package:flutter_plugin_tools/src/common/core.dart';
+import 'package:flutter_plugin_tools/src/common/plugin_command.dart';
+
+/// A command that runs integration test of plugin examples.
+class IntegrationTestCommand extends PluginCommand {
+  /// Creates an instance of the integration-test command, which runs the
+  /// integration test of each plugin example on physical targets or emulators.
+  IntegrationTestCommand(Directory packagesDir) : super(packagesDir) {
+    argParser.addFlag(
+      _generateEmulatorsArg,
+      help: 'Create and destroy emulators during test.\n\n'
+          'Must provide either $_platformsArg or $_recipeArg option to specify\n'
+          'which platforms to create.',
```

Comment:
Not just here.

Reply:
I've followed the style of [PluginCommand](https://github.com/flutter/plugins/blob/ec80d3439adb860d5f44c2d8427d459b10cda9d0/script/tool/lib/src/common/plugin_command.dart#L74-L93) since those messages will be shown together with ones in the IntegrationTestCommand. Also the output format is ugly when newlines are removed (Texts aren't wrapped well like the flutter tool).

flutter_tizen_plugin_tools, newlines removed:
![flutter_tizen_plugins_tools](https://user-images.githubusercontent.com/43136596/149468026-8c679d73-37bb-4569-a850-93a5334d6c73.png)

flutter:
![flutter tools](https://user-images.githubusercontent.com/43136596/149468077-536be05d-9af3-49f5-aa69-742e2f8bfdc6.png)

Reply:
It depends on the user's terminal size. I actually see no problem with the output in the first image.

One thing you could do is to change the help value of the platforms option (`<device_profile>-<platform_version>`) to something like just `profile-version` (without brackets).

I also tried setting the `usageLineLength` argument of `CommandRunner` but it didn't work.

Reply:
>I also tried setting the usageLineLength argument of CommandRunner but it didn't work.

I also tried this but it didn't seem to work as I expected. 

I'll remove the newline characters.